Develop

jponeticarvalho opened the pull request:
An stable version to be candidate to a 0.1 version

jponeticarvalho committed:
Adding the Allies table

Creating the Ally table.

Commit:
add rewardsAndIncentives.json

Commit:
translate rewardsAndIncentives.json

andreluizgaspar committed:
Merge branch 'jponeticarvalho:main' into main

Commit:
fix minor typos

Commit:
fix minor typos

jponeticarvalho committed:
Add the Allies information on contract

Add the allies info to the contract and add the allies info to the display.

jponeticarvalho committed:
Update Complication Table

Update the way we roll the table of complications.
Add the dices for each complication type

jponeticarvalho committed:
Update Antagonist Table

Update the way we roll the table of antagonist.
Add the dices for each antagonist type

jponeticarvalho committed:
Update Location Table

Update the way we roll the table of location.
Add the dices for each location type

jponeticarvalho committed:
Merge pull request #2 from TheAndreLuiz/main

add/translate rewardsAndIncentives.json

jponeticarvalho committed:
Update Objective Table

Update the way we roll the table of objective.
Add the dices for each objective type

jponeticarvalho committed:
Update Contractor/ContractType Table

Update the way we roll the table of Contractor/ContractType.
Add the dices for each Contractor/ContractType type

jponeticarvalho committed:
Update difficulty/distance Table

Update the way we roll the table of difficulty/distance.
Add the dices for each difficulty/distance type

jponeticarvalho committed:
Update dueDate Table

Update the way we roll the table of dueDate.
Add the dices for each dueDate type

jponeticarvalho committed:
Update Difficulty and Distance dices

Fix a bug where the dices isn't the corrects

jponeticarvalho committed:
Update the rollTable function position

jponeticarvalho committed:
Add the Extra Reward information on contract

Add the Extra Reward info to the contract and add the Extra Reward info to the display.

jponeticarvalho committed:
Add the Trunarounds information on contract

Add the Trunarounds info to the contract and add the Trunarounds info to the display.

jponeticarvalho committed:
Update the theme for the interface

jponeticarvalho committed:
Add the contractKeyword information on contract

Add the contractKeyword info to the contract and add the contractKeyword info to the display.

jponeticarvalho committed:
Add Unusual Contractors information on contract

Add the Unusual Contractors info to the contract and add the Unusual Contractors info to the display.

jponeticarvalho committed:
Add the contractorKeyword information on contract

Add the contractorKeyword info to the contract and add the contractorKeyword info to the display.

jponeticarvalho committed:
Update user interface and integrations with ChatGPT

A big update in the user interface, to a better experience.
Integration with chatGPT to generate the contract context

jponeticarvalho committed:
Remove unused files

jponeticarvalho committed:
Update README.md

jponeticarvalho committed:
Update the imports after remove the unused files

mateustoin commented on README.md:
Seria interessante melhorar um pouco o README, adicionando algumas imagens da tela e pelo menos 1 caso de uso da aplicação.

jponeticarvalho replied:
Including to TODO task's.
May not include in the PR.

mateustoin commented on config.json:
Talvez mudar o nome do arquivo para "Global configs" do app ou algo semelhante. Apenas "config" é algo genérico.

jponeticarvalho replied:
Accepted

mateustoin reviewed (approved):
I left some comments suggesting some things that you might want to see later.

jponeticarvalho committed:
Add confirmation dialogs when:

Close the application;
Delete the no existent guilds;
Restore to default configs.

jponeticarvalho committed:
Update the config file

Update the config file name;
Update the config file path on ContractviewerApp

jponeticarvalho committed:
Merge branch 'main' of https://github.com/jponeticarvalho/guild-rpg-maker into develop

jponeticarvalho committed:
Fix a bug where the guild list wasn't updated

mateustoin reviewed (approved):
Fixed.
